### DISCUSSION TITLE
Disable Control+CMD keyboard shortcut

### DISCUSSION THREAD

**Discussion by kettanaito** [rxhanson/RectanglePro-Community#605] [Q&A] [answered]
Hi! Thank you for the awesome tool. 

Rectangle Pro ships a `Ctrl+CMD` (+ click) shortcut that breaks the default Ctrl+CMD+D (translate) shortcut on macOS. How can I disable this shortcut? 

I've searched for it in Keyboard Shortcuts and any other tab in the tool that lists any shortcuts, haven't found any way to disable/remap it. Thanks.

**Comment by rxhanson** [comment, marked as answer]
Thanks for reporting. As a quick check, we can see if Rectangle Pro is executing anything for that shortcut in the logs. Hold the option key in the menu bar menu and select View Logging, then try the keyboard shortcut. 

The only other default setting that uses control + command is the window throw, but that's not intercepting a keyboard shortcut. You could try removing the modifier keys for the Window Throw and see if that makes a difference. 

As a side note, Control+Command+D works fine for me alongside Rectangle Pro.

**Reply by kettanaito** [reply]
Thanks for a quick reply!

In the View Logging terminal I can see this being printed when I press CTRL+CMD:

```
2024-12-09T17:39:41+01:00: Updating visible window info
```

It doesn't print any command name to be more detailed. 

> You could try removing the modifier keys for the Window Throw and see if that makes a difference.

This helped! 🎉 Thank you.
